### PR TITLE
SEO: fix wrong device button in preview on mobile

### PR DESCRIPTION
**Pull request by Copons**
Fixes: https://github.com/Automattic/wp-calypso/issues/9488.

On small screens, replaced the wrong phone preview button with the
correct SEO preview one.

| Before | After |
| --- | --- |
| screen shot 2016-11-19 at 18 52 46 | screen shot 2016-11-19 at 18 52 59 |

### CODE REVIEW

**Comment by drw158** [comment]
Design 👍

File: client/components/web-preview/toolbar.jsx [hunk 1]
```diff
@@ -81,15 +81,16 @@ export const PreviewToolbar = props => {
...
+			{ showDeviceSwitcher &&
 			<button
 				aria-hidden={ true }
 				key={ 'back-to-preview' }
 				className={ classNames(
 					'web-preview__device-button',
 					'web-preview__back-to-preview-button', {
-					'is-active': currentDevice !== 'seo'
-				} ) }
+						'is-active': currentDevice !== 'seo'
```
Comment (obenland) [inline client/components/web-preview/toolbar.jsx]: Minor, but would you want to make this yoda?

Comment (Copons) [inline client/components/web-preview/toolbar.jsx]: Sure!

**Comment by obenland** [comment]
🚢!

**Pull request merged**